crates/forge/tests/data/collection_with_lib/Scarb.toml

```
@@ -1,9 +1,12 @@
 [package]
 name = "collection_with_lib"
 version = "0.1.0"
-edition = "2023_10"
+edition = "2024_07"
 
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 
 [dependencies]
 # foo = { path = "vendor/foo" }
+
+[dev-dependencies]
+cairo_test = "2.9.2"
```

This should have dependency for `snforge_std` not `cairo_test`
```suggestion

[dev-dependencies]
snforge_std = { path = "../../../../../snforge_std" }

```